packages/html-reporter/src/links.tsx

```diff
@@ -77,9 +78,9 @@ export const AttachmentLink: React.FunctionComponent<{
   return <TreeItem title={<span>
     {attachment.contentType === kMissingContentType ? icons.warning() : icons.attachment()}
     {attachment.path && <a href={href || attachment.path} download={downloadFileNameForAttachment(attachment)}>{linkName || attachment.name}</a>}
-    {attachment.body && <span>{attachment.name}</span>}
+    {attachment.body && <span>{linkifyText(attachment.name)}</span>}
   </span>} loadChildren={attachment.body ? () => {
-    return [<div className='attachment-body'><CopyToClipboard value={attachment.body!}/>{attachment.body}</div>];
+    return [<div className='attachment-body'><CopyToClipboard value={attachment.body!}/>{linkifyText(attachment.body!)}</div>];
```

Comment:
```suggestion
    return [<div className='attachment-body'><CopyToClipboard value={attachment.body!}/>{attachmentBody}</div>];
```

let's do `const attachmentBody = attachment.body ? linkifyText(attachment.body) : null` to be on the safe side?

Reply:
There is a check already at line 82, it's just typescript is not smart enough